packages/block-editor/src/components/inserter-list-item/index.js

```diff
@@ -45,6 +45,7 @@ function InserterListItem( {
 					onClick();
 				} }
 				disabled={ isDisabled }
+				__experimentalIsFocusable
```

Comment:
I noticed that this kind of breaks the current keyboard navigation on the inserter. Pressing <kbd>Tab</kbd>, instead of moving to the next block group, moves focus to the disabled item:

![Pressing tab on the block inserter](https://user-images.githubusercontent.com/3068563/101910529-d0910000-3b9d-11eb-87f6-f3ce046fb37d.gif)

I think this happens because our `Button` will force `tabIndex={0}` on the element, whereas the roving tabindex approach on the block inserter requires it to be `tabIndex={-1}` when not focused.

This can be easily fixed by using the CompositeItem's `focusable` prop instead:

```suggestion
				focusable
```

Reply:
I couldn't see a `tabIndex` on the element when testing.

But that's a great catch, and switching to `focusable` works.